admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -1240,17 +1275,17 @@ protected function _default_tickets_update(EE_Event $evtobj, $data)
                 }
             }
             $DTM->save();
-            $DTT = $evtobj->_add_relation_to($DTM, 'Datetime');
+            $DTM = $evtobj->_add_relation_to($DTM, 'Datetime');
             // load DTT helper
             // before going any further make sure our dates are setup correctly so that the end date is always equal or greater than the start date.
-            if ($DTT->get_raw('DTT_EVT_start') > $DTT->get_raw('DTT_EVT_end')) {
-                $DTT->set('DTT_EVT_end', $DTT->get('DTT_EVT_start'));
-                $DTT = EEH_DTT_Helper::date_time_add($DTT, 'DTT_EVT_end', 'days');
-                $DTT->save();
+            if ($DTM->get_raw('DTT_EVT_start') > $DTM->get_raw('DTT_EVT_end')) {
+                $DTM->set('DTT_EVT_end', $DTM->get('DTT_EVT_start'));
+                $DTM = EEH_DTT_Helper::date_time_add($DTM, 'DTT_EVT_end', 'days');
+                $DTM->save();
             }
-            // now we got to make sure we add the new DTT_ID to the $saved_dtts array  because it is possible there was a new one created for the autosave.
-            $saved_dtt = $DTT;
-            $success = ! $success ? $success : $DTT;
+            // now we got to make sure we add the new DTT_ID to the $saved_dtts array
+            //  because it is possible there was a new one created for the autosave.
+            $saved_dtt = $DTM;
```

Comment:
THEN the same object as above is "copied" to the `$saved_dtt` var as well as to ...
 the`$success` var? which can also be a boolean ?!?!?